usercommandmgmt/usercommandmgmt.py

```diff
@@ -186,25 +193,138 @@ def get_highest_user_comm_allowance(self, member):
 
         return max(rel_usr_roles)
 
-    @staticmethod
-    def mod_evaluate_command(self, proposed_msg):
+    async def submit_for_approval(self, ctx, command, text):
         """
         posts the proposed command in the specified channel, and waits for a moderator's reaction to either
-        reject or accept the command. Will  return true if command_moderation is set to False or command
-        passes moderator evalutaion. False if it is rejected by a moderator.
-        Note: proposed_msg is the proposed command
+        reject or accept the command.
         """
-        if not command_moderation:
-            return True
-        else:
-            # submits command for evaluation and awaits response; TODO
-            pass
+        # create message to submit for mod approval
+        message_to_submit = (
+            "Sender: "
+            + ctx.author.name
+            + "\n"
+            + "Proposed Command: "
+            + "\n"
+            + command
+            + "\n"
+            + text
+            + "\n \n"
+            + ctx.message.jump_url
+        )
```

Comment:
This gets hard to read with the `+`s, and if in the future you decide to change it, then it might get messy. Consider making this a constant docstring at the top of the file and something similar to the following:
```
MESSAGE_TEMPLATE = """Sender: {author}
Proposed Command: {command}
{text}
{jump_url}
"""

...

message_to_submit = MESSAGE_TEMPLATE.format(
    author=ctx.author.name,
    command=command,
    text=text,
    jump_url=ctx.message.jump_url)
```

Reply:
Seconding what Injabie said, but also check out f strings. They're my personal preference for handling string stuff like this.

Reply:
Seconding .format